enterprise/frontend/src/embedding-sdk/cli/snippets/express-server-snippet.ts

```diff
@@ -0,0 +1,139 @@
+import { HARDCODED_USERS } from "embedding-sdk/cli/constants/hardcoded-users";
+
+import {
+  HARDCODED_JWT_SHARED_SECRET,
+  USER_ATTRIBUTE_CUSTOMER_ID,
+} from "../constants/config";
+
+interface Options {
+  instanceUrl: string;
+  tenantIds: (number | string)[];
+}
+
+const DEFAULT_EXPRESS_SERVER_PORT = 4477;
+
+export const getExpressServerSnippet = (options: Options) => {
+  const users = HARDCODED_USERS.map((user, i) => ({
+    ...user,
+
+    // Assign one of the tenant id in the user's database to their Metabase user attributes.
+    // This is hard-coded for demonstration purposes.
+    ...(options.tenantIds[i] && {
+      [USER_ATTRIBUTE_CUSTOMER_ID]: options.tenantIds[i],
+    }),
+  }));
+
+  return `
+const express = require("express");
+const fetch = require("node-fetch");
+const jwt = require("jsonwebtoken");
+const session = require('express-session')
+const cors = require('cors')
+
+const PORT = process.env.PORT || ${DEFAULT_EXPRESS_SERVER_PORT}
+
+const METABASE_INSTANCE_URL = '${options.instanceUrl}'
+
+const METABASE_JWT_SHARED_SECRET =
+  '${HARDCODED_JWT_SHARED_SECRET}'
+
+const USERS = ${JSON.stringify(users, null, 2)}
+
+const getUser = (email) => USERS.find((user) => user.email === email)
+
+async function metabaseAuthHandler(req, res) {
+  const { user } = req.session;
+
+  if (!user) {
+    return res.status(401).json({
+      status: "error",
+      message: "not authenticated",
+    });
+  }
+
+  const token = jwt.sign(
+    {
+      email: user.email,
+      first_name: user.firstName,
+      last_name: user.lastName,
+      groups: user.groups,
+      customer_id: user.customerId,
```

Comment:
This must be `customer_id` to match https://github.com/metabase/metabase/blob/697080616314a595c95afc8edc7a6b774aa4571f/enterprise/frontend/src/embedding-sdk/cli/constants/config.ts#L22

Sandboxing is broken without this.

Reply:
@WiNloSt Good eye! I must've replaced the hard-coded `customerId: ...` without noticing this. Let me actually replace `[customer_id]` with the `USER_ATTRIBUTE_CUSTOMER_ID` constant.